public/index.php

```diff
@@ -4,6 +4,26 @@
  * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
+if (ini_get('display_errors')) {
+    /*
+     * PHP by default returns HTTP 200 even when exceptions are thrown.
+     *
+     * Now by default return HTTP 500 so any API client can trust in the HTTP response code.
+     */
+    http_response_code(500);
```

Comment:
Also: Rasmus challenged me on this recently, and indicated that 500 is thrown for fatal errors (uncaught exceptions are converted to fatal errors). Could this be an issue of an extension such as XDebug changing the status?